bitnami/etcd/templates/scripts-configmap.yaml

```
@@ -10,6 +10,7 @@
 {{- $etcdPeerProtocol := include "etcd.peerProtocol" . }}
 {{- $etcdClientProtocol := include "etcd.clientProtocol" . }}
 {{- $initSnapshotFilename := .Values.startFromSnapshot.snapshotFilename }}
+{{- $snapshotHistoryLimit := .Values.cronjob.snapshotHistoryLimit }}
```

Variable in template is incorrect, because it is nested under `disasterRecovery`, below is code change suggestion (not tested, though):

```suggestion
{{- $snapshotHistoryLimit := .Values.disasterRecovery.cronjob.snapshotHistoryLimit }}
```